### ISSUE TITLE
Get travis-ci.com working with this repo

### ISSUE THREAD

**Issue by mpenkov** [RaRe-Technologies/gensim-wheels#15]
Our travis-ci.com are currently not running because of a billing issue:

https://www.travis-ci.com/github/RaRe-Technologies/gensim-wheels/requests

    Could not authorize build request for RaRe-Technologies/gensim-wheels.

@piskvorky It looks like we have to explicitly select a plan to use with these guys

https://stackoverflow.com/questions/41034694/travis-could-not-authorize-build-request/41078031

The free plan should be sufficient, IMHO.

We should also apply for OSS credits:

https://docs.travis-ci.com/user/billing-faq/#what-if-i-am-building-open-source

**Comment by piskvorky** [comment]
Okay. I selected the Free plan.

I'll can try to apply for the additional OSS credit if we need it – do we?

**Comment by mpenkov** [comment]
I think so. I think the free plan does not recharge, so it will eventually run out.

**Comment by mpenkov** [comment]
Yeah, it sounds like we'll definitely need OSS credits, cause our free plan already ran out. Geez...

> Builds have been temporarily disabled for public repositories due to a negative credit balance. Please go to the Plan page to replenish your credit balance or alter your Consume paid credits for OSS setting.
